common/src/main/java/fi/hel/verkkokauppa/common/queue/service/SendNotificationService.java

```diff
@@ -95,4 +94,37 @@ public void sendRefundMessageNotification(
 
     }
 
+    public void sendErrorNotification(
+            String endpoint,
+            String message,
+            Exception exception
+    ) {
+        sendErrorNotification(
+                endpoint,
+                message + " " + exception.getMessage(),
```

Comment:
Make endpoint and message to be one message parameter no need for duplicate strings.

Reply:
Made them separate so that it would be clear that endpoint should be given but can be combined

Reply:
But is there allways an endpoint what is called?